### PR TITLE
Add Symbiosis bridge

### PR DESCRIPTION
**Pull request by lucadonnoh**
prev #1572 

https://l2beat-staging-git-symbiosis-finance-master-l2beat.vercel.app/bridges/projects/symbiosis

How to add a preview?
https://www.notion.so/l2beat/New-project-preview-9a20e970b89b4f7da008349ab12ca00a

### CODE REVIEW

File: packages/config/src/bridges/symbiosis.ts [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+export const symbiosis: Bridge = {
+  type: 'bridge',
+  id: ProjectId('symbiosis'),
+  display: {
+    name: 'Symbiosis',
+    slug: 'symbiosis',
+    category: 'Hybrid',
+    description:
+      'Symbiosis is a cross-chain AMM DEX that pools together liquidity from different networks, whether they use EVM technology or not.',
```
Comment (antooni) [inline packages/config/src/bridges/symbiosis.ts:18]: rewrite the desciption, make it sound more professional, also include some info about how it is validated

File: packages/config/src/bridges/symbiosis.ts [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    links: {
```
Comment (antooni) [inline packages/config/src/bridges/symbiosis.ts:19]: maybe also link to the app?

**Review by antooni — APPROVED** [review]

File: packages/config/src/bridges/symbiosis.ts (hunk 1 above)
Comment (fastchain) [reply packages/config/src/bridges/symbiosis.ts:18]: Hi! 
Thank for the feedback.


`make it sound more professional`
Could you please provide a link to any example that sounds really professional?


`include some info about how it is validated`
I'm afraid I don't get it, what you mean by validated?   It works this way right now, so everyone can go to the app and validate it by his own hands.

Comment (lucadonnoh) [reply packages/config/src/bridges/symbiosis.ts]: i'll take care of it :)

File: packages/config/src/bridges/symbiosis.ts (hunk 2 above)
Comment (IngeFX) [reply packages/config/src/bridges/symbiosis.ts:19]: If you think it will be convenient to add it here, then: https://app.symbiosis.finance/

**Pull request merged**